ios/StatusIm/AppDelegate.m

```diff
@@ -101,4 +105,36 @@ - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
 #endif
 }
 
+- (void)applicationWillResignActive:(UIApplication *)application {
+  if ([[NSUserDefaults standardUserDefaults] boolForKey:@"BLANK_PREVIEW"]) {
+    // fill screen with our own colour
+    UIView *colourView = [[UIView alloc]initWithFrame:self.window.frame];
```

Comment:
I think it is better to just save it as a local variable than a view with a tag.

Reply:
done